### PR TITLE
Integrate spec: Milestone 10

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this milestone, the following was done:

- Use Capybara to write integration tests for each view in your project.
- Implemented Login page tests
- Implemented User index page tests
- Implemented user show page tests
- Implemented User post index page tests
- Implemented Post-show page tests

Please run the test with:

```rb
bundle exec rspec
```
test

### CODE REVIEW

File: spec/features/user/user_index_spec.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-user_index_spec.rb
+require 'spec_helper'
```
Comment (Davidosky007) [inline spec/features/user/user_index_spec.rb:1]: * I suggest you add Tests that asserts you can see the profile picture for each user.
* And also Tests that redirects to the user's show page when you click on a user.

File: spec/features/user/user_show_spec.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-user_show_spec.rb
+require 'spec_helper'
```
Comment (Davidosky007) [inline spec/features/user/user_show_spec.rb:1]: * I suggest You add all these tests:  Tests that asserts you can see the user's profile picture.
* Tests that redirects to the post's show page when you click on a user's post.
* Tests that asserts you can see the user's first 3 posts.

File: spec/features/post/post_index_spec.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+require 'spec_helper'
```
Comment (Davidosky007) [inline spec/features/post/post_index_spec.rb:1]: * Kindly add all these missing tests: Tests that asserts you can see the user's profile picture.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @pasytchangwa @eroiyo 👋, 

Good job so far! 💪 
* Professional looking readme file ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Proper use of git-flow ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-jumpin-gif


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/features/post/post_index_spec.rb (hunk 3 above)
Comment (pasytchangwa) [reply spec/features/post/post_index_spec.rb:1]: done

File: spec/features/user/user_show_spec.rb (hunk 2 above)
Comment (pasytchangwa) [reply spec/features/user/user_show_spec.rb:1]: done

File: spec/features/user/user_index_spec.rb (hunk 1 above)
Comment (pasytchangwa) [reply spec/features/user/user_index_spec.rb:1]: done

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**